libs/block-editor/src/lib/extensions/services/suggestions/suggestions.service.ts

```diff
@@ -19,7 +19,7 @@ export class SuggestionsService {
 
     getContentTypes(filter = ''): Observable<DotCMSContentType[]> {
         return this.http
-            .get(`/api/v1/contenttype?filter=${filter}&orderby=name&direction=ASC`, {
+            .get(`/api/v1/contenttype?filter=${filter}&orderby=name&direction=ASC&per_page=40`, {
```

Comment:
I would need to confirm with @wezell about this one because I think the 40 only applies is there is a filter applied.

Reply:
Confirmed, 40 and force to filter more.